### ISSUE TITLE
VERB has two nsubj

### ISSUE THREAD

**Issue by hg-sun** [explosion/spaCy#3768]
Hi, I was trying to get dep, pos from below example and found out a root VERB has two nsubj. Linguistically, I think it is impossible to have two nsubj(phone with index 2 and version with index 
 9) from same VERB (have with index 12)but is it possible or it is just a bug?
 
<pre><code>>>>import spacy
>>>nlp = spacy.load('en_core_web_sm')
>>>test = "An excellent phone to have but unfortunately the unlocked version doesn't have it's standard video calling"
>>>doc =  nlp(test)
>>>for token in doc:
>>>    print(token, token.pos_, token.dep_, token.head, token.head.i)
An DET det phone 2
excellent ADJ amod phone 2
phone Noun nsubj have 12
to PART aux have 4
have VERB relcl phone 2
but CCONJ cc phone 2
unfortunately ADV advmod have 12
the DET det version 9
unlocked ADJ amod version 9
version NOUN nsubj have 12
does VERB aux have 12
't ADV neg have 12
have VERB ROOT have 12
it PRON subj 's 14
's PART ccomp have 12
standard ADJ amod calling 17
calling NOUN attr 's 14
</code></pre>

## Your Environment

* Operating System: Linux-4.4.0-130-generic-x86_64-with-debian-stretch-sid
* Python Version Used:3.6.5
* spaCy Version Used:2.1.3
* Environment Information:

**Comment by BramVanroy** [comment]
First of all, that does not look like a valid English sentence. The _it's standard_ should probably be left out. Lingusiticaly, it does make sense to have two subjects if you think about it in an implicit coreference kind-of way. 

`[An excellent phone][1] to have but unfortunately [the unlocked version][1'] doesn't have video calling`

**Comment by hg-sun** [comment]
Thanks for the reply.

Since above sentence was crawled in online review site, it is unlikely that it is a valid English sentence.
Beside that, I tried the sentence that you wrote in both console and web (https://explosion.ai/demos/displacy), the result case differently.

In console, same as before, [phone] and [version] were both connected with same [have] with same dependency; nsubj, but in the web, [An excellent phone] was treated as a phrase then, the phrase was connected with [have] by 'conj'. The result on the web was what I was expecting.

I used both with same en_model (en_core_web_sm 2.0.0).
